### PR TITLE
SHOR-123: Style Civisualise extension

### PR DESCRIPTION
**Pull request by igorpavlov**
## Overview

This PR styles the Civisualise extension https://civicrm.org/extensions/civisualize-missing-data-visualization-extension.

## Technical Details

```css
/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */

.page-civicrm-dataviz {
  #datacount {
    margin-top: $crm-standard-gap;

  // we are styling the same as Civi Dashboard panels
  div.dc-chart:not(#datacount),
  [class^='col-md-'][id] {
    background-color: $crm-white;
    border-radius: $border-radius-child;
    margin-bottom: #{$crm-standard-gap / 2};
    margin-right: #{$crm-standard-gap / 2};
    overflow: hidden;
    text-align: center;

    > strong:first-of-type,
    h3 {
      background-color: $gray-dark;
      color: $crm-white;
      display: block;
      font-size: $font-size-medium;
      line-height: auto;
      padding: #{$crm-standard-gap / 2} $crm-standard-gap;

    g.row text,
    .pie-slice {
      // white text on white background is not visible, so... black
      // note: black charts backgrounds are not gonna happen
      fill: $crm-black;

    h3 + .dc-chart {
      margin: $crm-standard-gap / 2;

  // tables also need treatment, we make them consistent with graphs
  table.dc-chart {
    background-color: $crm-white;
    border-collapse: collapse;
    border-radius: $border-radius-child;
    overflow: hidden;

    th,
    td {
      border: 0;
      padding: #{$crm-standard-gap / 2} $crm-standard-gap;

    thead th {
      background-color: $gray-dark;
      color: $crm-white;

    tbody tr {
      border-bottom: solid 1px $gray-light;

    .header-fixed {
      position: relative;

  // Until we have Bootstrap, we have to use this rule to hide
  // unwanted elements, we cannot accept showing them.
  .hidden,
  .hide {
    display: none;
```

## Tests

Only manual tests were performed hence the styling is isolated to specific pages.

### CODE REVIEW

**Comment by jamienovick** [comment]
@igorpavlov can we just check that none of the charts are getting clipped by the headers? On the events it looks like it might be. Thanks!

**Comment by igorpavlov** [comment]
> @igorpavlov can we just check that none of the charts are getting clipped by the headers? On the events it looks like it might be. Thanks!

I do not think this is due to styling, this is how extension works:

_Vanilla version_
image

**Comment by jamienovick** [comment]
lol, noted...

**Comment by igorpavlov** [comment]
@jamienovick done 
image

**Comment by jamienovick** [comment]
Boom! Hero. thanks!

File: scss/civicrm/contact/_contact.scss [hunk 1]
```diff
@@ -47,3 +47,4 @@ $page-class-name: str_slice($page-class, 2);
 @import 'pages/change-log';
 @import 'pages/pledges';
 @import 'pages/find-contacts';
+@import 'pages/dataviz';
```
[the remark this answers is not in the document]
Comment (igorpavlov) [reply scss/civicrm/contact/_contact.scss]: The main identifying class was `page-....-dataviz` - that's why I called it like that. I agree, referencing the extension name is much better.

**Pull request merged**